keyboards/planck/keymaps/snowkuma/custom_keycodes.h

```diff
@@ -0,0 +1,64 @@
+// These definitions are for convenience.
+// It is not wise to put sensitive information here such as passwords
+// as anyone with access to your keyboard will be able to use them!
+
+// magnet keycodes
+#define M_LEFT LCA(KC_LEFT)
+#define M_TOP LCA(KC_UP)
+#define M_BOTT LCA(KC_DOWN)
+#define M_RGHT LCA(KC_RGHT)
+
+#define M_TOPL LCA(KC_U)
+#define M_TOPR LCA(KC_I)
+#define M_BOTL LCA(KC_J)
+#define M_BOTR LCA(KC_K)
+
+#define M_L13 LCA(KC_D)
+#define M_L23 LCA(KC_E)
+#define M_C13 LCA(KC_F)
+#define M_R23 LCA(KC_T)
+#define M_R13 LCA(KC_G)
+
+#define M_NEXT LCAG(KC_RGHT)
+#define M_PREV LCAG(KC_LEFT)
+
+#define M_MAX LCA(KC_ENT)
+#define M_CEN LCA(KC_C)
+#define M_REST LCA(KC_BSPC)
+
+// Shortcuts
+#define INPUT_L LCAG(KC_SPC)
+#define TXT_PLS LGUI(KC_PLUS)
+#define TXT_MIN LGUI(KC_MINS)
+#define SC_CAPF LGUI(LSFT(KC_3))  // Capture the full screen to file
+#define SC_CAPP LGUI(LSFT(KC_4))  // Capture portion of screen to file
+ 
+// Convenience Keycodes
```

Comment:
QMK defines aliases for these keycodes automatically. If you use the already-defined aliases, Lines 36-38 can be deleted.

https://github.com/qmk/qmk_firmware/blob/a120a000d481ed14a87988383be46c5a76604bb6/quantum/quantum_keycodes.h#L29-L31